grafana-plugin/src/plugin.json

```diff
@@ -33,64 +33,64 @@
       "type": "page",
       "name": "Alert Groups",
       "path": "/a/grafana-oncall-app/?page=incidents",
-      "role": "Viewer",
+      "action": "grafana-oncall-app.alert-groups:read",
       "addToNav": true,
       "defaultNav": true
     },
     {
       "type": "page",
       "name": "Users",
       "path": "/a/grafana-oncall-app/?page=users",
-      "role": "Viewer",
+      "action": "grafana-oncall-app.user-settings:read",
       "addToNav": true
     },
     {
       "type": "page",
       "name": "Integrations",
       "path": "/a/grafana-oncall-app/?page=integrations",
-      "role": "Viewer",
+      "action": "grafana-oncall-app.integrations:read",
       "addToNav": true
     },
     {
       "type": "page",
       "name": "Escalation Chains",
       "path": "/a/grafana-oncall-app/?page=escalations",
-      "role": "Viewer",
+      "action": "grafana-oncall-app.escalation-chains:read",
       "addToNav": true
     },
     {
       "type": "page",
       "name": "Schedules",
       "path": "/a/grafana-oncall-app/?page=schedules",
-      "role": "Viewer",
+      "action": "grafana-oncall-app.schedules:read",
       "addToNav": true
     },
     {
       "type": "page",
       "name": "ChatOps",
       "path": "/a/grafana-oncall-app/?page=chat-ops",
-      "role": "Viewer",
+      "action": "grafana-oncall-app.chatops:read",
       "addToNav": true
     },
     {
       "type": "page",
       "name": "Outgoing Webhooks",
       "path": "/a/grafana-oncall-app/?page=outgoing_webhooks",
-      "role": "Viewer",
+      "action": "grafana-oncall-app.outgoing-webhooks:read",
       "addToNav": true
     },
     {
       "type": "page",
       "name": "Maintenance",
       "path": "/a/grafana-oncall-app/?page=maintenance",
-      "role": "Viewer",
+      "action": "grafana-oncall-app.maintenance:read",
       "addToNav": true
     },
     {
       "type": "page",
       "name": "Settings",
       "path": "/a/grafana-oncall-app/?page=settings",
-      "role": "Viewer",
+      "action": "grafana-oncall-app.other-settings:read",
```

Comment:
You can leave the role, if RBAC is disabled, we will enforce the role but if it's enabled, we will use the action.